rocky/rocky/middleware/onboarding.py

```diff
@@ -9,7 +9,8 @@ def OnboardingMiddleware(get_response):
     def middleware(request):
         response = get_response(request)
         if request.user.is_authenticated:
-            member_onboarded = filter(lambda o: o.onboarded, request.user.organization_members)
+            member_onboarded = list(filter(lambda o: o.onboarded, request.user.organization_members))
```

Comment:
I think we should move this to be a property on the organisationMember, which should be a property of the user available trough request.
So available as:
request.user.member_onboarded

Which is loaded when needed, and cached for the duration of the request.

Reply:
`request.user.organization_members` is already a cached property of `KATUser`; what advantage would moving this line to a property have?

Reply:
I like keeping data local to where other related data lives. In this case, this member_onboarded flag is an abstraction of the data on the organization_members list and should be available from that data. It makes the API cleaner, and makes sure there's no need to reinvent this code if we want to know if a user is onboarded anywhere else.
come to think of it, it should just be a property on the user object, not of the organization_members property.

Reply:
I agree this would be cleaner API. It would also be better to use `any(o.onboarded for o in organization_members)` which will result in a boolean instead of making a list.

Reply:
If we want to know if an `OrganizationMember` is onboarded, we can simply look at its `onboarded` property.
`member_onboarded` is a list of onboarded `OrganizationMember`s, which we only use to look if the current user is onboarded on any organization (we only onboard the user on the first organization?).

We could make this a boolean flag and move it to the `KATUser` if we agree that's better?